### PR TITLE
request to add kubestellar-devs group

### PR DESCRIPTION
**Pull request by clubanderson**
changes to ownership and strategy for kcp is requiring kubestellar (fka kcp-edge)to change strategy and establish our own slack channel, mailing list, and etc.  We would like to continue working within the kubernetes ecosystem as we are expanding our support to work with a variety of different kubernetes distributions, not just kcp.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by clubanderson** [comment]
@jdumars, @markyjackson-taulia I updated the PR with the GH handles for the 4 'owners' of the kcp-edge-devs community.  PROW is complaining that it does not know these handles.  Where must we register so that we can be recognized by PROW?

**Comment by markyjackson-taulia** [comment]
@clubanderson Copy member ID. It will be a 9 character string of uppercase letters and numbers (example: U1H63D8SZ).

https://github.com/kubernetes/community/blob/master/communication/slack-guidelines.md (towards the bottom of the page)

**Comment by clubanderson** [comment]
ok - I updated users.yaml and will wait for changes to take affect.  Thank you

**Comment by clubanderson** [comment]
@markyjackson-taulia can I get an update on this?

**Comment by clubanderson** [comment]
can someone explain the error coming from test 'pull-community-tempelis-check'?

**Comment by clubanderson** [comment]
@markyjackson-taulia can I get an update on this?

**Comment by markyjackson-taulia** [comment]
I will look at this in the am @clubanderson

**Comment by MadhavJivrajani** [comment]
@clubanderson -- you will also have to update [users.yaml](https://github.com/kubernetes/community/blob/master/communication/slack-config/users.yaml) with the mapping of the members' GitHub ID -> Slack Member ID (see [here](https://github.com/kubernetes/community/blob/master/communication/slack-guidelines.md#requesting-a-user-group) for steps).

Here's a reference PR: https://github.com/kubernetes/community/pull/7130

File: communication/slack-config/usergroups.yaml
Comment (MadhavJivrajani) [inline communication/slack-config/usergroups.yaml:296]:
```suggestion
    description: "Active developers of KubeStellar on #kubestellar-dev"
```
Since `#kubestellar-dev` will be considered as a comment and not actually displayed when viewed on slack.
(probably needs to be fixed for the `kcp-devs` handle too)

**Comment by clubanderson** [comment]
@MadhavJivrajani I have committed your suggestion for changes and check out #7277 for the users.yaml.  Can you lgtm #7277?

File: communication/slack-config/usergroups.yaml
Comment (markyjackson-taulia) [inline communication/slack-config/usergroups.yaml:299]: @clubanderson for giggles, can you makes this all lowercase?

Comment (clubanderson) [reply communication/slack-config/usergroups.yaml]: done

**Comment by clubanderson** [comment]
@markyjackson-taulia I converted all IDs to lowercase

**Comment by markyjackson-taulia** [comment]
@coderanger can you see what is wrong here?

**Comment by clubanderson** [comment]
I had PROW names in here - not slack IDs from Kube slack org.  Fingers crossed.

**Comment by clubanderson** [comment]
this might be depended on #7277?

**Comment by clubanderson** [comment]
Yes. Names are in kube slack

**Comment by MadhavJivrajani** [comment]
@clubanderson thanks for persisting through the reviews. While https://github.com/kubernetes/community/pull/7277 merges, could you please clean up the commit history here?
For future reference -- it might be easier to modify both files in a single PR (easy to track changes too)

**Comment by MadhavJivrajani** [comment]
Looks like we're good to go here

**Comment by MadhavJivrajani** [comment]
Let's merge this after https://github.com/kubernetes/community/pull/7276

**Review by MadhavJivrajani — APPROVED** [review]
/lgtm
/approve
/hold cancel

**Pull request merged**
